app/controllers/application_controller.rb

```diff
@@ -95,8 +95,8 @@ def render_not_found
     end
   end
 
-  def render_forbidden
-    render plain: "forbidden", status: :forbidden
+  def render_forbidden(error = "forbidden")
+    render plain: error, status: :forbidden
```

Comment:
I'd be interested in knowing the reason for the change; it seems possible now to pass in a different string from "forbidden" but still give an HTTP forbidden status.

Reply:
It was based on this [suggestion](https://github.com/rubygems/rubygems.org/pull/3155#discussion_r938946247) of extracting forbidden renders to its own function in the API base controller. It appears that there was one already in the application controller that we can reuse.